### ISSUE TITLE
Feature: Add command line flag to specify location of usort config

### ISSUE THREAD

**Issue by thatch** [facebookexperimental/usort#31]
By default, µsort will automatically look for the "nearest" pyproject.toml for each file being formatted, and load the config from that location before formatting the associated file. For some projects, it would be preferable to specify the location of a canonical pyproject.toml that will be used as configuration for all files being formatted, regardless of any other pyproject.toml found near these files. This would also enable loading configs from a generic toml file, but we should always expect to find config values in the PEP-518 compliant `tool.usort` table.

The flag should probably be something like:
```shell-session
$ usort --config path/to/config.toml format ...
```

Normally, `usort.sorting.usort_path()` and `usort.sorting.usort_stdin()` call `Config.find(...)` to locate pyproject.toml and receive an appropriate Config object. A solution for this should probably set a flag or call a classmethod on Config to override that behavior and always supply the preferred configuration.

**Comment by thatch** [comment]
@jreese I think you were going to deprecate all command line flags except for overriding config location?  Do you want to reword this to match that goal?

**Comment by jreese** [comment]
Updated title and summary.
